### ISSUE TITLE
Only display the field "Who can ignore the task" field if the task if required, and move it

### ISSUE THREAD

**Issue by andergmartins** [publishpress/PublishPress-Checklists#283]
The "Who can ignore the task" field only makes sense if the task is required. In order to free space in the table we can move it to inside the cell of the "Disabled, Recommended or Required" field:

<img width="1117" alt="Screen Shot 2021-06-11 at 16 04 31" src="https://user-images.githubusercontent.com/407131/121737392-2bfd6580-cacf-11eb-97ba-84dc4768cf20.png">

**Comment by stevejburge** [comment]
@andergmartins good idea. This is related https://github.com/publishpress/PublishPress-Checklists/issues/274

**Comment by andergmartins** [comment]
If we decide to go "positive" as suggested on #274 this idea stops making sense (I'm not saying it is bad), but the field gains meaning for both "Required" and "Recommended" options. So I think we probably should decide later for one or another.

**Comment by stevejburge** [comment]
@andergmartins Sounds good. We can circle back to this
